src/Debug/MailerCollector.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Mailer\Debug;
+
+use Yiisoft\Mailer\MessageInterface;
+use Yiisoft\Yii\Debug\Collector\CollectorTrait;
+use Yiisoft\Yii\Debug\Collector\SummaryCollectorInterface;
+
+final class MailerCollector implements SummaryCollectorInterface
+{
+    use CollectorTrait;
+
+    private array $messages = [];
+
+    public function collectMessage(
+        MessageInterface $message,
+    ): void {
```

Comment:
```suggestion
    public function collect(
        MessageInterface $message,
        MessageInterface ...$messages,
    ): void {
```

Reply:
Why?

Reply:
This signature combines two methods into one.